Searcher.java

```diff
@@ -0,0 +1,41 @@
+import java.util.*;
+import java.io.*;
+import java.util.regex.*;
+
+/**
+ * Class for searching a file for a regex pattern.
+ *
+ */
+public class Searcher {

```

Comment:
Maybe have all code in the same class, since the program is small.

Reply:
Id argue that it is better to have two, since we can use the Searcher class in more places than one.

Reply:
Ok